source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -62,6 +62,18 @@ def _get_detailsSummary(self) -> typing.Optional[str]:
 	def hasDetails(self) -> bool:
 		return bool(self.IA2Attributes.get("details-roles"))
 
+	def _get_detailsRole(self) -> typing.Optional[controlTypes.Role]:
+		# Currently only defined in Chrome as of May 2022
+		# Refer to ComputeDetailsRoles
+		# https://chromium.googlesource.com/chromium/src/+/main/ui/accessibility/platform/ax_platform_node_base.cc#2419
+		detailsRoles = self.IA2Attributes.get("details-roles")
+		if not detailsRoles:
+			if config.conf["debugLog"]["annotations"]:
+				log.debug("no details-roles")

```

Comment:
This is a misleading log message.

Reply:
how so? I've updated it to "details-roles not found"